src/signalo/core/models.py

```diff
@@ -22,13 +26,50 @@ def _serialized(self):
         return f'{{"id": "{str(self.id)}", "type": "Feature", "geometry": {self.geom.geojson}, "properties": {{"name": "{self.name}"}}}}'
 
 
+@register_oapif_viewset()
+class Azimuth(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    value = models.SmallIntegerField(default=0, null=False, blank=False)
+    pole = models.ForeignKey(
+        Pole, on_delete=models.CASCADE, blank=True, null=True, related_name="azimuths"
+    )
+
+
+@receiver(signals.pre_delete, sender=Azimuth)
```

Comment:
wouldn't this be post delete?
shall we integrate the tests in the same PR?

Reply:
I just want to have a reliable and agreed upon basis to do meaningful work. The purpose of this is just to validate the relations are correct. If you can confirm that the relations are indeed correct, we can even close this.